console-framework-client-api/src/main/java/io/axoniq/console/framework/api/clientIdentification.kt

```diff
@@ -116,6 +116,7 @@ data class CommandBusInformation(
         val handlerInterceptors: List<InterceptorInformation> = emptyList(),
         val dispatchInterceptors: List<InterceptorInformation> = emptyList(),
         val messageSerializer: SerializerInformation?,
+        val multiTenant: Boolean = false,
```

Comment:
Unfortunately for decoding this with jackson-kotlin this isn't enough :( 
```suggestion
        val multiTenant: Boolean? = false,
```

If you don't do it like this, we get no creator exceptions, unfortunately.